far/vmenu.cpp

```diff
@@ -2357,6 +2484,18 @@ void VMenu::DrawTitles() const
 
 		Text(concat(L' ', string_view(strBottomTitle).substr(0, WidthTitle), L' '));
 	}
+
+#ifdef _DEBUG
```

Comment:
Is it needed?

Reply:
Not absolutely, but it is very helpful while debugging or simply understanding how the block of items moves horizontally. I'd slightly prefer leaving it here, but do not mind removing it if you so say.

Reply:
Maybe wrap it in `if constexpr ((false)) { ... }` then?
We have this pattern in a few places already.
This way it can still be easily switched on if needed and remains syntax-checked in all configurations.

Reply:
Good point, fixed.

Why double parenthesis by the way? I understand the difference they make (expression vs. literal), but I still do not see why. [Cppreference](https://en.cppreference.com/w/cpp/language/if) seems to agree:
```C++
void f()
{
    if constexpr(false)
    {
        int i = 0;
        int *p = i; // Error even though in discarded statement
    }
}
```

Reply:
> Why double parenthesis by the way?

Just a convention to stop the compiler from complaining about unreachable code.
https://godbolt.org/z/3G9zhbqqq

Reply:
Interesting. MSVC seems to be happy.
https://godbolt.org/z/97nhxq4sn

Reply:
_Now_ Sonarcloud is [unhappy](https://sonarcloud.io/project/issues?resolved=false&pullRequest=789&id=FarGroup_FarManager&open=AY2Z1Ptaj2xfvbCEaoYK). 🤣